docs/sources/setup/migrate/ssd-to-distributed/_index.md

```diff
@@ -0,0 +1,372 @@
+       chunk_encoding: snappy
+   ruler:
+    enable_api: true
+    storage:
+      type: s3
+      s3:
+        region: eu-central-1
+        bucketnames: aws-ruler-bucket
+        s3forcepathstyle: false
+      alertmanager_url: http://prom:9093
+   querier:
+      max_concurrent: 4
+
+   storage:
+      type: s3
+      bucketNames:
+        chunks: "aws-chunks-bucket"
+        ruler: "aws-ruler-bucket"
+      s3:
+        region: eu-central-1
+
+deploymentMode: SimpleScalable
+
+# SSD
+backend:
+ replicas: 2
+read:
+ replicas: 3
+write:
+ replicas: 3
+
+# Distributed Loki
+ingester:
+ replicas: 0
+ zoneAwareReplication:
+  enabled: false
+
+querier:
+ replicas: 0
+ maxUnavailable: 0
+queryFrontend:
+ replicas: 0
+ maxUnavailable: 0
+queryScheduler:
+ replicas: 0
+distributor:
+ replicas: 0
+ maxUnavailable: 0
+compactor:
+ replicas: 0
+indexGateway:
+ replicas: 0
+ maxUnavailable: 0
+ruler:
+ replicas: 0
+ maxUnavailable: 0
+
+# Single binary Loki
+singleBinary:
+ replicas: 0
+
+minio:
+ enabled: false
+```
+
+{{< admonition type="note" >}}
+This example is only a reference on the parameters that need to be changed. There will be other parameters within your own config such as `limits_config`, `gateway`, `compactor`, etc. These can remain the same.
+{{< /admonition >}}
+
+## Stage 1: Deploying the Loki distributed components
+
+In this stage, we will deploy the distributed Loki components alongside the SSD components. We will also change the `deploymentMode` to `SimpleScalable<->Distributed`. The `SimpleScalable<->Distributed` migration mode allows for a zero-downtime transition between Simple Scalable and fully Distributed architectures. During migration, both deployment types run simultaneously, sharing the same object storage backend.
+
+```
+Simple Scalable Components   ->    Distributed Components
+------------------------    ->    ---------------------
+write (StatefulSet)         ->    Distributor + Ingester
+read (Deployment)           ->    Query Frontend + Querier
+backend (StatefulSet)       ->    Compactor + Ruler + Index Gateway
+```
+Request routing:
+
+The Gateway (nginx) handles request routing based on endpoint type:
+1. Write Path (`loki/api/v1/push`):
+   * Initially routes to Simple Scalable write component
+   * Gradually shifted to the Distributor
+   * Both write paths share the same object storage, ensuring data consistency
+2. Read Path (`/loki/api/v1/query`):
+   * Routes to either Simple Scalable read or Distributed Query Frontend
+   * Query results are consistent since both architectures read from same storage
+3. Admin/Background Operations:
+   * Compaction, retention, and rule evaluation handled by either backend or respective distributed components
+   * Operations are coordinated through object storage locks
+
+To start the migration process:  
+
+1. Create a copy of your existing `values.yaml` file and name it `values-migration.yaml` (You can also make these changes within your current values file).
```

Comment:
If they keep the same file name, then the command at line 254 would fail, wouldn't it?
```suggestion
1. Create a copy of your existing `values.yaml` file and name it `values-migration.yaml`.
```